### PR TITLE
add transaction definition

### PR DESCRIPTION
**Pull request by imatwawana**
Part of tasks for [issue 4270](https://github.com/getsentry/sentry-docs/issues/4270)

### CODE REVIEW

File: src/docs/product/performance/transaction-summary.mdx [hunk 1]
```diff
@@ -8,6 +8,20 @@ Every transaction has a summary view that gives you a better understanding of it
 
 ![Example of Transaction Summary](./perf-transaction-summary.png)
 
+## What Is a Transaction?
+
+A transaction represents a single instance of an activity you want to measure or track, like a page load, page navigation, or an asynchronous task. Having transaction information lets you monitor the overall performance of your application beyond when it crashes or generates an error. Without transactions, you can only know when things in your application have actually gone wrong — which is important, but not the whole picture.
+
+Transactions on the Performance page are grouped by the transaction names, such as an endpoint paths for backend request transactions (like `/store/checkout/` or `api/v2/users/<user_id>/`) or URLs/routes for page load transactions (like `https://docs.sentry.io/performance-monitoring/`). When you click a transaction on the **Performance** page, it takes you to the **Transaction Summary**, where you can see a list of transaction events: that is, the individual instances of that specific transaction.
```
Comment (wmak) [inline src/docs/product/performance/transaction-summary.mdx:15]: Is it more grammatically correct to say `such as an endpoint's path`?

**Review by wmak — APPROVED** [review]

File: src/docs/product/performance/transaction-summary.mdx (hunk 1 above)
Comment (imatwawana) [reply src/docs/product/performance/transaction-summary.mdx:15]: yep good catch!

Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:15]:
```suggestion
Transactions on the Performance page are grouped by the transaction names, such as endpoint paths for backend request transactions (like `/store/checkout/` or `api/v2/users/<user_id>/`) or URLs/routes for page load transactions (like `https://docs.sentry.io/performance-monitoring/`). When you click a transaction on the **Performance** page, it takes you to the **Transaction Summary**, where you can see a list of transaction events: that is, the individual instances of that specific transaction.
```

File: src/docs/product/performance/transaction-summary.mdx [hunk 2]
```diff
@@ -8,6 +8,20 @@ Every transaction has a summary view that gives you a better understanding of it
...
+
+<Note>
+
+Learn more about setting transaction names in our SDK documentation [platform redirect link here](/platform-redirect/?next=/enriching-events/transaction-name/).
```
Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:19]:
```suggestion
Learn more about setting transaction names in our [SDK documentation](/platform-redirect/?next=/enriching-events/transaction-name/).
```

File: src/docs/product/performance/transaction-summary.mdx [hunk 3]
```diff
@@ -8,6 +8,20 @@ Every transaction has a summary view that gives you a better understanding of it
...
+
+</Note>
+
+For a more in-depth explanation of what’s in a transaction, as it relates to traces, check out our [full tracing documentation](/product/sentry-basics/tracing/).
```
Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:23]:
```suggestion
For a more in-depth explanation of what’s in a transaction as it relates to traces, check out our [full tracing documentation](/product/sentry-basics/tracing/).
```

File: src/docs/product/performance/transaction-summary.mdx [hunk 4]
```diff
@@ -8,6 +8,20 @@ Every transaction has a summary view that gives you a better understanding of it
 
 ![Example of Transaction Summary](./perf-transaction-summary.png)
 
+## What Is a Transaction?
+
+A transaction represents a single instance of an activity you want to measure or track, like a page load, page navigation, or an asynchronous task. Having transaction information lets you monitor the overall performance of your application beyond when it crashes or generates an error. Without transactions, you can only know when things in your application have actually gone wrong — which is important, but not the whole picture.
+
+Transactions on the Performance page are grouped by the transaction names, such as endpoint paths for backend request transactions (like `/store/checkout/` or `api/v2/users/<user_id>/`) or URLs/routes for page load transactions (like `https://docs.sentry.io/performance-monitoring/`). When you click a transaction on the **Performance** page, it takes you to the **Transaction Summary**, where you can see a list of transaction events: that is, the individual instances of that specific transaction.
```
Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:15]:
```suggestion
Transactions on the **Performance** page are grouped by the transaction names, such as endpoint paths for backend request transactions (like `/store/checkout/` or `api/v2/users/<user_id>/`) or URLs/routes for page load transactions (like `https://docs.sentry.io/performance-monitoring/`). When you click a transaction on the **Performance** page, it takes you to the **Transaction Summary**, where you can see a list of transaction events: that is, the individual instances of that specific transaction.
```

**Pull request merged**